drivers/overlay/overlay.go

```diff
@@ -2220,11 +2223,12 @@ func (d *Driver) ApplyDiffWithDiffer(id, parent string, options *graphdriver.App
 		differOptions.UseFsVerity = graphdriver.DifferFsVerityIfAvailable
 	}
 	out, err := differ.ApplyDiff(applyDir, &archive.TarOptions{
-		UIDMaps:           idMappings.UIDs(),
+		ForceMask:         forceMask,
 		GIDMaps:           idMappings.GIDs(),
 		IgnoreChownErrors: d.options.ignoreChownErrors,
-		WhiteoutFormat:    d.getWhiteoutFormat(),
 		InUserNS:          unshare.IsRootless(),
+		UIDMaps:           idMappings.UIDs(),
```

Comment:
Absolutely non-blocking: `UIDMaps` and `GIDMaps` are now far from each other, I’m not sure why this change was made.

(Personally I’d aesthetically prefer the fields to be in the same order as in the structure declaration, to make it easier to see which were set / omitted.)

Reply:
I've sorted them alphabetically.  I'll revert this part and only add `ForceMask`